bigquery/tests/unit/test_job.py

```diff
@@ -3341,8 +3341,8 @@ def test_from_api_repr_normal(self):
 
 class _Table(object):
```

Comment:
This class actually seems completely unnecessary now that the real `Table` class never makes an API call. I've filed https://github.com/GoogleCloudPlatform/google-cloud-python/pull/5126 to remove it.